README.md

```diff
@@ -41,6 +41,7 @@ Sentry is a developer-first error tracking and performance monitoring platform t
 - [Elixir](https://github.com/getsentry/sentry-elixir)
 - [Unity](https://github.com/getsentry/sentry-unity)
 - [Unreal Engine](https://github.com/getsentry/sentry-unreal)
+- [PowerShell](https://github.com/getsentry/sentry-powershell)
```

Comment:
I'm unable to test any of the changes in this PR because the `devenv sync` gets stuck consistently with message `Waiting for docker to be ready.... (timeout in 90s)`